src/app/products/products-routing.module.ts

```diff
@@ -1,9 +1,27 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { BrandsComponent } from './brands/brands.component';
+
+import { InventoryCountsComponent } from './inventory-counts/inventory-counts.component';
+import { PriceBookComponent } from './price-book/price-book.component';
+import { ProductTagsComponent } from './product-tags/product-tags.component';
+import { ProductTypesComponent } from './product-types/product-types.component';
 
 import { ProductsComponent } from './products.component';
+import { PromotionsComponent } from './promotions/promotions.component';
+import { StockControlComponent } from './stock-control/stock-control.component';
+import { SuppliersComponent } from './suppliers/suppliers.component';
 
-const routes: Routes = [{ path: '', component: ProductsComponent }];
+const routes: Routes = [{ path: '', component: ProductsComponent },
```

Comment:
repeatition here

Reply:
but as im seeing only one lyk in my code im not seeing the repetition

Reply:
that's okay then